### PR TITLE
Scrolling Issue Fix

### LINKED ISSUE #43: Scroll gracefully when more batches than height of screen

### PR DESCRIPTION
**Pull request by beausmith**
Fixes #43

Ideally we update QR Code scanning to be handled with a solution other than using a text input (such as a window listener), but until then, this is a fine solution.

### CODE REVIEW

**Comment by beausmith** [comment]
FYI @benadida  - This issue was introduced with [feature to allow manual scanning via a USB QR code scanner](https://github.com/votingworks/bsd/commit/0db7af9745169afc9f8ce2d3983b5688780d7fb5). The layout breakage happens when the autofocus to the text input causes the page to scroll to the offscreen hidden input element.

File: src/App.tsx [hunk 1]
```diff
@@ -198,9 +203,14 @@ const App: React.FC = () => {
...
 
   useInterval(() => {
-    election && updateStatus()
-    cardServerAvailable && !election && readCard()
-    document.getElementById('ballotString')!.focus()
+    if (election) {
+      updateStatus()
+      ballotScannerInput &&
+        ballotScannerInput.current &&
+        ballotScannerInput.current.focus()
```
Comment (eventualbuddha) [inline src/App.tsx]: Looking forward to optional chaining!

```ts
ballotScannerInput?.current?.focus()
```

Just waiting on React…

Comment (beausmith) [reply src/App.tsx]: Ferserious!

**Comment by beausmith** [comment]
@benadida - I guess I prematurely returned the barcode scanner to you; I had forgotten about this feature. 

Because your originally wrote this feature, do you want to review this PR?

**Comment by benadida** [comment]
@beausmith No you can kill this feature. Hand scanner was a backup that isn't appropriate for our new binary qr codes anyways.

**Comment by beausmith** [comment]
(╯°□°) ╯︵ ┻━┻

Lol! Will update PR with more red. 😁

**Comment by benadida** [comment]
oh wait, I just saw the whole thread. I think you're saying *I* broke it. And you're right. I'm so, so sorry.

**Comment by benadida** [comment]
@beausmith confirming that you're handling the removal of the feature?

**Comment by beausmith** [comment]
Yes, I'm on this!

**Comment by beausmith** [comment]
@benadida - ready for one final review from you.

**Pull request merged**